cmd/nodecmd/loadtest.go

```diff
@@ -0,0 +1,302 @@
+		}
+		if existingSeparateInstance == "" {
+			for _, sg := range filteredSGList {
+				if err = grantAccessToPublicIPViaSecurityGroup(ec2SvcMap[sg.region], loadTestNodeConfig.PublicIPs[0], sg.securityGroup, sg.region); err != nil {
+					return err
+				}
+			}
+		}
+	case constants.GCPCloudService:
+		var gcpClient *gcpAPI.GcpCloud
+		var gcpRegions map[string]int
+		var imageID string
+		var projectName string
+		if existingSeparateInstance == "" {
+			// Get GCP Credential, zone, Image ID, service account key file path, and GCP project name
+			gcpClient, gcpRegions, imageID, _, projectName, err = getGCPConfig(true)
+			if err != nil {
+				return err
+			}
+			regions := maps.Keys(gcpRegions)
+			separateHostRegion = regions[0]
+			loadTestCloudConfig, err = createGCPInstance(gcpClient, nodeType, map[string]int{separateHostRegion: 1}, imageID, clusterName, true)
+			if err != nil {
+				return err
+			}
+			loadTestNodeConfig = loadTestCloudConfig[separateHostRegion]
+		} else {
+			_, projectName, _, err = getGCPCloudCredentials()
+			if err != nil {
+				return err
+			}
+			loadTestNodeConfig, separateHostRegion, err = getNodeCloudConfig(existingSeparateInstance)
+			if err != nil {
+				return err
+			}
+		}
+		if !useStaticIP {
+			loadTestPublicIPMap, err := gcpClient.GetInstancePublicIPs(separateHostRegion, loadTestNodeConfig.InstanceIDs)
+			if err != nil {
+				return err
+			}
+			loadTestNodeConfig.PublicIPs = []string{loadTestPublicIPMap[loadTestNodeConfig.InstanceIDs[0]]}
+		}
+		if existingSeparateInstance == "" {
+			if err = grantAccessToPublicIPViaFirewall(gcpClient, projectName, loadTestNodeConfig.PublicIPs[0], "loadtest"); err != nil {
+				return err
+			}
+		}
+	default:
+		return fmt.Errorf("cloud service %s is not supported", cloudService)
+	}
+	if existingSeparateInstance == "" {
+		if err := saveExternalHostConfig(loadTestNodeConfig, separateHostRegion, cloudService, clusterName); err != nil {
+			return err
+		}
+	}
+	var separateHosts []*models.Host
+	separateHostInventoryPath := filepath.Join(app.GetAnsibleInventoryDirPath(clusterName), constants.MonitoringDir)
+	if existingSeparateInstance == "" {
+		if err = ansible.CreateAnsibleHostInventory(separateHostInventoryPath, loadTestNodeConfig.CertFilePath, cloudService, map[string]string{loadTestNodeConfig.InstanceIDs[0]: loadTestNodeConfig.PublicIPs[0]}, nil); err != nil {
+			return err
+		}
+	}
+	separateHosts, err = ansible.GetInventoryFromAnsibleInventoryFile(separateHostInventoryPath)
+	if err != nil {
+		return err
+	}
+
+	if err := GetLoadTestScript(app); err != nil {
+		return err
+	}
+
+	// waiting for all nodes to become accessible
+	if existingSeparateInstance == "" {
+		failedHosts := waitForHosts(separateHosts)
+		if failedHosts.Len() > 0 {
+			for _, result := range failedHosts.GetResults() {
+				ux.Logger.PrintToUser("Instance %s failed to provision with error %s. Please check instance logs for more information", result.NodeID, result.Err)
```

Comment:
nit: use ux.Logger.RedXToUser

Reply:
I suggest using RedXToUser every time we try to use ux.Logger.PrintToUser before returning err